modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -242,6 +252,26 @@ export default class LargeVideoManager {
                 const tracks = state['features/base/tracks'];
                 const videoTrack = getVideoTrackByParticipant(tracks, participant);
 
+                // Remove track streaming status listener from the old track and add it to the new track,
+                // in order to stop updating track streaming status for the old track and start it for the new track.
+                // TODO: when this class is converted to a function react component,
+                // use a custom hook to update a local track streaming status.
+                if (this.videoTrack?.jitsiTrack?.getSourceName?.() !== videoTrack?.jitsiTrack?.getSourceName?.()) {
```

Comment:
`getSourceName?.()` doesn't need the `?` (in both arguments)

Reply:
changed